### ISSUE TITLE
Unable to install gcloud sdk on M1 Macbook

### ISSUE THREAD

**Issue by ptantiku** [google-github-actions/setup-gcloud#329]
### TL;DR
Unable to install gcloud sdk on M1 Macbook when trying to run locally (using "[act](https://github.com/nektos/act)") because the code generates wrong download URL.

**Expected behavior**
gcloud successfully installed and the actions continues

**Observed behavior**
The action stops with `::error::error code: 404`. No other information.
![image](https://user-images.githubusercontent.com/479656/122955804-a325da00-d3aa-11eb-8eec-92d0c2ccc88d.png)

**Additional information**
After investigated and found that it generates the URL in this format `https://dl.google.com/dl/cloudsdk/channels/rapid/downloads/google-cloud-sdk-${version}-darwin-${arch}.tar.gz`
([source](https://github.com/google-github-actions/setup-gcloud/blob/master/setupGcloudSDK/src/format-url.ts#L40)).
When using on M1, `arch == 'arm64'` so it generates a URL ending with `-darwin-arm64.tar.gz`.

But, on [Google SDK download links](https://cloud.google.com/sdk/docs/quickstart#mac), the URL ends with `-darwin-arm.tar.gz`. This is the cause of `error code: 404`.
![image](https://user-images.githubusercontent.com/479656/122953039-9b653600-d3a8-11eb-814f-2ead684af55d.png)

**Comment by bharathkkb** [comment]
@ptantiku thanks for investigating. I have opened a bug internally to track and hoping to resolve this upstream. 

x-ref: b/191835428

**Comment by bharathkkb** [comment]
https://github.com/google-github-actions/setup-cloud-sdk/pull/35 adds support in the SDK
@averikitsch Did you have another tracking issue for switching to use `setup-cloud-sdk` in `setup-gcloud`?
